scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala

```diff
@@ -73,19 +83,26 @@ object TypedOsv extends TypedSeperatedFile {
 }
 
 object FixedPathTypedDelimited {
-  def apply[T: Manifest: TupleConverter: TupleSetter](path: String, separator: String): FixedPathTypedDelimited[T] =
+  def apply[T: TypeDescriptor](path: String, separator: String): TypedTextDelimited[T] =
     apply(Seq(path), separator)
 
-  def apply[T: Manifest: TupleConverter: TupleSetter](paths: Seq[String], separator: String): FixedPathTypedDelimited[T] = {
-    val f = Dsl.intFields(0 until implicitly[TupleConverter[T]].arity)
-    apply(paths, f, separator)
+  def apply[T: TypeDescriptor](paths: Seq[String], separator: String): TypedTextDelimited[T] = {
+    new FixedTypedText(source.TypedSep(separator), paths: _*)
   }
 
-  def apply[T: Manifest: TupleConverter: TupleSetter](path: String, f: Fields, separator: String): FixedPathTypedDelimited[T] =
+  def apply[T: TypeDescriptor](path: String, f: Fields, separator: String): TypedTextDelimited[T] =
     apply(Seq(path), f, separator)
 
-  def apply[T: Manifest: TupleConverter: TupleSetter](paths: Seq[String], f: Fields, separator: String): FixedPathTypedDelimited[T] =
-    new FixedPathTypedDelimited[T](paths, f, false, false, separator)
+  def apply[T: TypeDescriptor](paths: Seq[String], f: Fields, separator: String): TypedTextDelimited[T] = {
```

Comment:
This would require having `TypedSeperatedDelimited.apply` call the `FixedPathTypedDelimited.apply` method.

Since we're planning to deprecate FixedPathTypedDelimited and TypedDelimited, this means we'll get deprecation warnings. Is that okay?